var/spack/repos/builtin/packages/gpi-space/package.py

```diff
@@ -66,10 +67,10 @@ class GpiSpace(CMakePackage):
     depends_on(
         "boost@1.62.0:1.63.0"
         "+atomic +chrono +coroutine +context +date_time +filesystem +iostreams"
-        " +program_options +random +regex +serialization +test +timer cxxstd=14"
+        " +program_options +random +regex +serialization +test +timer cxxstd=17"
```

Comment:
Isn't this change breaking wrt previous versions of the package?

Reply:
Oh, yes, look like you a re right. I am going to learn how to tell spack to use different options for different versions, can you provide a pointer to the relevant piece of documentation? That would be very helpful, thanks.

Reply:
Fine to keep history, we usually squash-merge PRs anyhow :slightly_smiling_face: